soh/include/z64player.h

```diff
@@ -608,7 +608,7 @@ typedef struct Player {
     /* 0x083E */ s16        targetYaw;
     /* 0x0840 */ u16        unk_840;
     /* 0x0842 */ s8         meleeWeaponAnimation;
-    /* 0x0843 */ s8         swordState; // Upstream TODO: meleeWeaponState
+    /* 0x0843 */ s8         meleeWeaponState; // Upstream TODO: meleeWeaponState
```

Comment:
Looks like we can drop the upstream todo
```suggestion
    /* 0x0843 */ s8         meleeWeaponState;
```